### DISCUSSION TITLE
更新通知（2017-04-22）

### DISCUSSION THREAD

**Discussion by SeaHOH** [SeaHOH/GotoX#25] [公告]
刚修复了一个影响比较大的错误，此错误可能导致某些转发链接无法关闭。
建议更新到以下修改：
https://github.com/SeaHOH/GotoX/tree/a9672c9ed9897c0f4b1cd2e2c3c9e04728058269

Clone 到本地，然后解压覆盖更新即可，**覆盖时注意保留之前的设置**。
已经更新过的请忽略。

**Comment by Just2co** [comment]
@SeaHOH 你好，我今晚测试GotoX一晚上，发现有很严重的问题，不知道是不是我哪里没设置好？
比如：打开谷歌系列网站，速度很快，每次打开一个其他的需翻墙的新网站，要卡半天，但是一旦打开了几次，后面再开同一个网站的其他页面，就会飞快，下载速度可达3MB每秒，尤其是图片比较多的网站。但是再换其他网站，又开始卡半天，不知道是不是DNS解析比较慢的问题？这个软件自带的IP列表，所有IP都是首尾连到一起的，中间没有用|隔开吗？我用第三方IP测试工具测试好之后导出来的格式，每个IP之间都是|隔开的，放到里面，打开软件测试IP也没有报错啊？还请帮我解答一下，为什么开谷歌外的网页会卡半天？要怎么处理才能搞定这个问题？没理由卡半天啊

**Comment by jzp820927** [comment]
我觉得可以下个红头文件。比较正规一点。
这个应该是缓存的原因。后面打开的，都是调用缓存的，肯定快了。

**Comment by SeaHOH** [comment]
@Just2co 
这种无关本 issuse 主题的问题最好单发个 issuse，或在你先前开的那个 issuse 提。

新开网站卡很可能是 DNS 解析的问题。
注意看配置文件，我估计是你的系统 DNS 本来就慢或者不能解析当前访问的域名。
你可以更换系统 DNS，或调整优先级使用另两种解析方式。
```ini
[dns]
# DNS 的优先级别，失败了才使用下一个，默认 system|remote|overhttps
# system：操作系统
# remote：即上面 server 中指定的
# overhttps：谷歌 DNS-over-HTTPS API
priority = system|overhttps|remote
```
新开网站卡还有一个原因就是，你没有把要翻墙的网站加入自动规则中，GotoX 会先尝试直连，失败后才会临时加入 GAE 规则 15 分钟。你可以看日志。

至于 IP 列表，就是每行一个 IP。
如果利用「记事本」看估计会连成一串，**请不要用「记事本」编辑本软件中的任何文件**。
请使用 notepad+、notepad2 等其它文本编辑工具。

**Comment by caofengzi** [comment]
怎麼把翻牆網站加入自動規則裡面?

**Comment by SeaHOH** [comment]
自己先看 ActionFilter.ini 的注释，不懂再问。
还有，不是说了这种无关本 issuse 主题的问题最好单发个 issuse 嘛。
